src/server/index.ts

```diff
@@ -234,7 +234,7 @@ export default (spreeClient: Instance) => {
     })
   })
 
-  app.listen(8889, () => {
-    logger.info('API listening on port 8889')
+  app.listen(process.env.SERVER_PORT, () => {
```

Comment:
Can we send options to the module function, like we did with `spreeOptions` and and `elasticSearchOptions`?